src/components/toolbar/toolbar.scss

```diff
@@ -42,6 +49,8 @@ md-toolbar {
 }
 
 .md-toolbar-tools {
+  @extend .md-title;
+  font-weight: 400;
```

Comment:
It depends–do we want component styles to stay in component `scss` or consolidate things into the typography file? It could go either way.